bot/modules/nostr/index.ts

```diff
@@ -0,0 +1,40 @@
+require('websocket-polyfill'); // is it needed?
```

Comment:
websocket-polyfill is needed for nodejs connect with websocket, if you have doubts try removing it and test if it works locally